ports/metis/portfile.cmake

```diff
@@ -4,9 +4,9 @@ set(OPTIONS -DSHARED=OFF)
 set(METIS_VERSION 5.1.0)
 
 vcpkg_download_distfile(ARCHIVE
-    URLS "http://glaros.dtc.umn.edu/gkhome/fetch/sw/metis/metis-${METIS_VERSION}.tar.gz"
-    FILENAME "metis-${METIS_VERSION}.tar.gz"
-    SHA512 deea47749d13bd06fbeaf98a53c6c0b61603ddc17a43dae81d72c8015576f6495fd83c11b0ef68d024879ed5415c14ebdbd87ce49c181bdac680573bea8bdb25
+    URLS "https://mirrors.edge.kernel.org/debian/pool/main/m/metis/metis_5.1.0.dfsg.orig.tar.xz"
+    FILENAME "metis_5.1.0.tar.xz"
+    SHA512 77d662dfade6131e17b934ba1b7f02e41dec701cfb0e0f0afce420e3a02489611200e323e288d88cd8de004b4d491c6a115ce0a9f9a0df1caba9f5682c67f2f0

```

Comment:
It would be good to sill use "${METIS_VERSION}" and keep a comment to explain that we use a more reliable mirror than the original repository with the link to it.